### PR TITLE
docs(forms): fix incorrect variables naming in the comments

### PR DESCRIPTION
**Pull request by denjamal**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

The comments for the source code contains incorrect variable naming 

Issue Number: N/A

## What is the new behavior?
The comments for the source code contains correct variable naming 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/forms/src/model.ts [hunk 1]
```diff
@@ -846,25 +846,25 @@ export abstract class AbstractControl {
  * ```ts
  * const ctrl = new FormControl('Nancy');
```
Comment (brandonroberts) [inline packages/forms/src/model.ts]: Update `ctrl` to `control` for consistency with other API pages

File: packages/forms/src/model.ts [hunk 2]
```diff
@@ -846,25 +846,25 @@ export abstract class AbstractControl {
...
+ * ctrl.reset('Drew');
  *
- * console.log(control.value); // 'Drew'
+ * console.log(ctrl.value); // 'Drew'
  * ```
  *
  * ### Reset the control back to an initial value and disabled
  *
  * ```
  * const ctrl = new FormControl('Nancy');
```
Comment (brandonroberts) [inline packages/forms/src/model.ts]: Same here

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by denjamal** [comment]
Hi @brandonroberts, I've finished the requested changes.

**Review by brandonroberts — APPROVED** [review]

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**